test/Tests.hs

```diff
@@ -7,14 +7,15 @@ import TypeCheckerTests
 
 -- collects all tests together
 spielTests :: Test
-spielTests = TestList [parserTests,evalTests]
+spielTests = TestList [parserTests,evalTests,typeCheckerTests]
 
 -- run all tests in the suite
 main :: IO ()
 main =  do
-  result     <- runTestTT spielTests  -- Run standard HUnit tests
-  parseResult<- checkParseAllExamples -- Custom test: parse all examples, return Pass | (Fail #failures)
-  tcResult   <- typeCheckAllExamples  -- Custom test: tc all examples, return Pass | (Fail #failures #errors)
+  result      <- runTestTT spielTests  -- Run standard HUnit tests
+  parseResult <- checkParseAllExamples -- Custom test: parse all examples, return Pass | (Fail #failures)
+  tcResult    <- typeCheckAllExamples  -- Custom test: tc all examples, return Pass | (Fail #failures #errors)
+  illTCResult <- typeCheckIll  -- type check ill typed examples (note: intentionally display only)
```

Comment:
This is part of unfinished work. Since this test case fails and we know it, I just display it in the output but do not make it so that it actually fails the test suite.

Reply:
That's fine for now. We'll have to work it out later anyways.